### PR TITLE
Adding to_payload_bytes and from_payload_bytes for OutgoingCommand and IncomingCommand respectively

### PR DESCRIPTION
**Pull request by evandp**

### CODE REVIEW

File: simplebgc/src/commands/motors_off.rs [hunk 1]
```diff
@@ -9,3 +12,31 @@ pub enum MotorsOffMode {
...
     SafeStop,
 }
+
+#[derive(BgcPayload, Copy, Clone, Debug, PartialEq)]
+pub struct MotorsOffQuery {
+    #[kind(enumeration)]
+    #[name("")]
+    #[format(u8)]
+    pub mode: MotorsOffMode,
+}
```
Comment (laptou) [inline simplebgc/src/commands/motors_off.rs]: If this is an unnamed property, shouldn't we just make this a tuple struct?

File: simplebgc/src/commands/mod.rs [hunk 2]
```diff
@@ -34,7 +34,7 @@ pub enum OutgoingCommand {
     BoardInfo3,
     Control(ControlData),
     MotorsOn,
-    MotorsOff { mode: Option<MotorsOffMode> },
+    MotorsOff { mode: Option<MotorsOffQuery> },
```
Comment (laptou) [inline simplebgc/src/commands/mod.rs]: Make this a tuple variant (`MotorsOff(MotorsOffQuery)`) for consistency

**Review by laptou — CHANGES_REQUESTED** [review]

File: simplebgc/src/commands/motors_off.rs (hunk 1 above)
Comment (evandp) [reply simplebgc/src/commands/motors_off.rs]: done

File: simplebgc/src/commands/mod.rs (hunk 2 above)
Comment (evandp) [reply simplebgc/src/commands/mod.rs]: done

**Review by laptou — APPROVED** [review]
LGTM

**Pull request merged**